recipes/alembic/all/patches/1.8.5-0001-fix-cmake.patch

```diff
@@ -0,0 +1,27 @@
+--- a/lib/Alembic/CMakeLists.txt
++++ b/lib/Alembic/CMakeLists.txt
+@@ -68,7 +68,7 @@ IF (ALEMBIC_SHARED_LIBS)
+ 
+   if (ALEMBIC_USING_IMATH_3)
+     if (APPLE OR UNIX)
+-      set_target_properties(Alembic PROPERTIES INSTALL_RPATH "$<TARGET_FILE_DIR:${ALEMBIC_ILMBASE_LIBS}>")
++      # set_target_properties(Alembic PROPERTIES INSTALL_RPATH "$<TARGET_FILE_DIR:${ALEMBIC_ILMBASE_LIBS}>")
```

Comment:
```suggestion
```
A very-very minor suggestion: prefer simply removing lines instead of commenting-out in patches. Makes for smaller patches and easier to immediately see what was changed.